.github/workflows/main.yml

```diff
@@ -88,8 +90,16 @@ jobs:
         with:
           go-version: '~${{ env.GO_VERSION }}'
 
-      - name: compile
-        run: make unit pkg=... case=_NONE_
+      - name: fetch and rebase on master
+        run: |
+          git remote add upstream https://github.com/lightningnetwork/lnd
+          git fetch upstream
+          export GIT_COMMITTER_EMAIL="lnd-ci@example.com"
+          export GIT_COMMITTER_NAME="LND CI"
+          git rebase upstream/master
```

Comment:
I assume this exits with non-zero code if rebasing fails and aborts the whole check?
Great to make sure you don't push anything that has conflicts in the first place :tada:

Reply:
Yes it should fail. Even if it doesn't, it would also fail later as there'd still be an in-progress rebase when running with `--exec`